### DISCUSSION TITLE
[C/C++] Spawned Process Error with Semmle

### DISCUSSION THREAD

**Discussion by vabui** [github/securitylab#404] [Q&A] [answered]
I am getting the following error when running Semmle with Guardian:

[2021-06-30 17:38:26] [ERROR] Spawned process exited abnormally (code 41; tried to run:

[C:\CerberusSourceBuildInPipelineAgent\_work\4\s\.gdn\i\nuget\microsoft.semmle.1.26.1\odasa\tools\runner.exe, cmd.exe,
 /C, type, NUL, &&, odasa, index, --auto, C:\Keil_v5\UV4\UV4.exe, -b, C:\CerberusSourceBuildInPipelineAgent\_work\4\s\Project-Cerberus\cerberus\Keil\Cerberus.uvoptx, -j0, -l, C:\CerberusSourceBuildInPipelineAgent\_work\4\s\Project-Cerberus\cerberus\Keil\Objects\logfile2.txt]

The compilation task does compile multiple projects. Is it possible that Semmle does not support analysis where multiple projects are compiled? Or does Semmle not support the Keil compiler?

**Comment by rdmarsh2** [comment, marked as answer]
Hi @vabui
We do support the Keil compiler and multiple projects, and this may be an issue with your pipeline setup. However, Guardian is an internal Microsoft tool that we don't support through this repo. Can you contact me from your Microsoft email at rdmarsh2@github.com? I can connect you to the right people to work through this issue.
